ops/ansible/group_vars/all/vars_file.yml

```diff
@@ -0,0 +1,3 @@
+---
+remote_user: "ubuntu"
```

Comment:
Should we have a step in the ansible playbook create a user defined by that variable and go from there? That's doable.